spec/support/revision_shared_examples.rb

```diff
@@ -77,7 +77,7 @@
         expect(page).not_to have_content item_type.to_s
       end
       expect(model.class.find_by_id(model.id)).not_to be_nil
-      expect(page).to have_content 'Recovered'
+      expect(page).to have_content(/recovered/i)
```

Comment:
do we have to use a regex? I mean, it is upcase or downcase, right? and we know it. Using the right format should just work?